### PR TITLE
fix-missing-technologies

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR fixes missing technologies in individual cards in catalog

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: _includes/partials/compatibility.html [hunk 1]
```diff
@@ -1,31 +1,32 @@
     <div class="comatibility-grid">
-    {% for item in pattern.compatibility %}
+    {% for items in pattern.compatibility %}
+    {% if items %}
+    {% for item in items %}
     {% if item == "Istio" %}
     <img class="service-mesh-image" src="{{site.baseurl}}/assets/images/istio.svg" />
-    {% endif %}
-    {% if item == "Linkerd" %}
+    {% elsif item == "Linkerd" %}
```
Comment (sudhanshutech) [inline _includes/partials/compatibility.html:7]: @vishalvivekm are these values are not available from api?

**Review by sudhanshutech — COMMENTED** [review]
What is left here?

@theBeginner86 do we need to hardcode those values

**Comment by sudhanshutech** [comment]
@vishalvivekm merge conflicts..

**Comment by sudhanshutech** [comment]
@vishalvivekm this is still needed?

**Comment by vishalvivekm** [comment]
> @vishalvivekm this is still needed?

Nope, Sudhanshu.

It's been implemented and merged  by @theBeginner86 : https://github.com/meshery/meshery.io/commit/d1f9406b2c978930cdf44a7e57ebdfe0b185275d

**Pull request closed without merging**